examples/rag/graph_rag_example.py

```diff
@@ -32,7 +33,7 @@ def _create_kg_connector():
     """Create knowledge graph connector."""
     return VectorStoreConnector(
         vector_store_type="KnowledgeGraph",
-        vector_store_config=VectorStoreConfig(
+        vector_store_config=BuiltinKnowledgeGraphConfig(
```

Comment:
`BuiltinKnowledgeGraphConfig` is not compatible with the paramter in `VectorStoreConnector.__init__`, refractor to `IndexStoreConnector` will solve this problem.

Reply:
but `BuiltinKnowledgeGraphConfig` is a subclass of `IndexStoreConnector ` too